src/solvers/refinement/string_constraint_generator_valueof.cpp

```diff
@@ -13,6 +13,17 @@ Author: Romain Brenguier, romain.brenguier@diffblue.com
 
 #include <solvers/refinement/string_refinement_invariant.h>
 #include <solvers/refinement/string_constraint_generator.h>
+#include <util/simplify_expr.h>
+
+#include <math.h>
+
+const exprt get_radix_from_optional_second_argument(
+  const function_application_exprt &f, const typet &type)
+{
+  return f.arguments().size()==1?
+    static_cast<exprt>(from_integer(10, type)):
```

Comment:
It doesn't compile without them. `from_integer` gives a constant_exprt, and the compiler complained if the branches of the `?` operator have different types.